data-node/src/test/java/org/graylog/datanode/testinfra/DatanodeContainerizedBackend.java

```diff
@@ -113,7 +114,7 @@ private NodeContainerConfig createConfig() {
     }
 
     private static ImageFromDockerfile createDockerImageFile(String opensearchVersion) {
-        final String opensearchTarArchive = "opensearch-" + opensearchVersion + "-linux-x64.tar.gz";
+        final String opensearchTarArchive = "opensearch-" + opensearchVersion + "-linux-" + OpensearchDistribution.archCode(System.getProperty("os.arch")) + ".tar.gz";
```

Comment:
I think we should also add a check that the tar archive exists. This can save us a lot of time if not and provide a valuable feedback even before the container tries to start. Any failure later is far more complicated to detect.